src/core/utils.js

```diff
@@ -19,10 +20,23 @@ export function camelCaseProps(obj) {
   return newObj;
 }
 
-export function getClientConfig(config) {
+export function getClientConfig(_config) {
   const clientConfig = {};
-  for (const key of config.get('clientConfigKeys')) {
-    clientConfig[key] = config.get(key);
+  for (const key of _config.get('clientConfigKeys')) {
+    clientConfig[key] = _config.get(key);
   }
   return clientConfig;
 }
+
+export function getAsBool(key, { _config = config } = {}) {
+  const value = _config.get(key);
```

Comment:
Ok, I suppose the benefit of that is it makes it clearer where the value is coming from.